src/router/routes/exportRoute.ts

```diff
@@ -16,18 +22,32 @@ export default class ExportRoute {
 
     private exportHandler: any;
 
-    constructor(bulkDataAccess: BulkDataAccess, authService: Authorization) {
+    private fhirVersion: FhirVersion;
+
+    private authService: Authorization;
+
+    constructor(bulkDataAccess: BulkDataAccess, authService: Authorization, fhirVersion: FhirVersion) {
         this.router = express.Router();
+        this.fhirVersion = fhirVersion;
+        this.authService = authService;
         this.exportHandler = new ExportHandler(bulkDataAccess, authService);
         this.init();
     }
 
     async initiateExportRequests(req: express.Request, res: express.Response, exportType: ExportType) {
+        const allowedResourceTypes = await this.authService.getAllowedResourceTypesForOperation({
+            operation: 'read',
+            userIdentity: res.locals.userIdentity,
+            requestContext: res.locals.userIdentity,
+        });
```

Comment:
we will be calling this for system exports too -- will be changing it's existing functionality -- from if admin be able to export everything to only export based on what scopes you provided; or in RBAC what resources align with said group

I think this will make the export more aligned with AuthZ but we will need a major version bump here and clear README explaining the change

Reply:
Gotcha. Will update the commit message on merge to: 

feat!: add support for group data export

BREAKING CHANGE: System export will only include resources based on what scopes you provided(SMART) or what resources align with said group(RBAC)